.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea/
```

Comment:
I won't be so strong with it. 

It has no impact to add this to gitignore, plus in some project you want to version certain (but not all file of `./idea`).

Having those file to gitignore global is nice:
```
# User-specific stuff:
.idea/**/workspace.xml
.idea/**/tasks.xml
.idea/dictionaries

# Sensitive or high-churn files:
.idea/**/dataSources/
.idea/**/dataSources.ids
.idea/**/dataSources.xml
.idea/**/dataSources.local.xml
.idea/**/sqlDataSources.xml
.idea/**/dynamic.xml
.idea/**/uiDesigner.xml
```

Having the whole `.idea` should be project specific.

What do you think @Minishlink ?